esp/esp/qsd/models.py

```diff
@@ -200,6 +200,9 @@ def program_from_url(url):
     def get_absolute_url(self):
         return "/"+self.url+".html"
 
+    class Meta:
+        verbose_name = 'Editable'
```

Comment:
We may want to look over the LU and MIT ESP wikis to make sure that there aren't references to "Quasi Static Datas on the admin panel".